examples/hosts/app-integration/external-data/src/mock-service-interface/dataParsing.ts

```diff
@@ -0,0 +1,37 @@
+/*!
```

Comment:
These contents were moved into `mock-service-interface` directory since it represents the interop data format used by the service.